pyoffers/models/offer_file.py

```diff
@@ -0,0 +1,26 @@
+from .core import Model, ModelManager
+
+
+class OfferFile(Model):
+
+    generic_methods = ('update', 'delete')
+
+
+class OfferFileManager(ModelManager):
+    model = OfferFile
+    name = 'offer_files'
+    generic_methods = (
+        'find_by_id',
+        'find_all',
+        'find_all_ids',
+        'delete',
+        'update',
+    )
+
+    def create(self, **kwargs):
+        assert 'filename' in kwargs, \
+            'You must specify ``filename`` argument, which should be a name of a readable file in current directory. ' \
+            'If the file is in the different directory, specify its path via ``local_path``.'
+        file_name = kwargs.get('filename')
```

Comment:
Just use a single variable for the file path. It will work in the same way as in any popular library or even in python itself.